src/blog/2024/07/customise-theming-in-your-dashboards.md

```diff
@@ -0,0 +1,215 @@
+
+### Left Side of the Header
+
+To render elements on the left side of the header, you can utilize the div element having the `#app-bar-title` ID, in which our page name is displayed.
+
+![Left Side Area](./images/left-side-area.png){data-zoomable}
+
+The `#app-bar-title` currently spans the full width of the header. When adding elements, they will occupy this area and align from left to right. However, if you add multiple elements in succession (using the default `display: flex`), they may extend beyond the width of the page view.
+
+#### Example: Adding Buttons
+
+1. Drag the ui-template widget onto the canvas.
+2. Double-click on it and select the scope to either `ui-scope` or `page-scope`. Selecting `ui-scope` will eliminate the need for multiple template widgets for the same purpose. For example, where you want the same buttons on all pages of your dashboard.
+3. Choose the page on which you want to render the buttons if you selected `page-scope`, or choose correct ui if `ui-scope` is selected.
+4. Paste the following Vue snippet into the template widget. In this snippet, note how we specify the "to" attribute targeting the `#app-bar-actions` ID in the teleport tag:
+
+```js
+<template>
+    <!-- Teleport the button to the #app-bar-actions area when mounted -->
+    <Teleport v-if="mounted" to="#app-bar-actions">
+        <v-btn>Button 1</v-btn>
+        <v-btn>Button 2</v-btn>
+        <v-btn>Button 3</v-btn>
+    </Teleport>
+</template>
+
+<script>
+    export default {
+        data() {
+            return {
+                mounted: false
+            }
+        },
+        mounted() {
+            // Set mounted to true when the component is mounted
+            this.mounted = true
+        }
+    }
+</script>
+```
+
+5. Next, you can customize further by adding more buttons or different elements inside the `<Teleport>` element.
+
+![Left Side Area](./images/button-added-dashboard.png){data-zoomable}
+
+#### Example: Adding Logo
+
+If you want to add your brand's logo, you can replace the element inside <teleport> with an <img> tag. You can do this in the same ui-template widget or in a different ui-template widget:
+
+1. Drag the ui-template widget onto the canvas.
+2. Select the correct scope for that widget to render.
+3. Select the correct page or UI in which you want to render the element.
+3. Paste the same Vue snippet given in the above section into the `ui-template` widget and replace the code inside <teleport> with the following element:
+
+```html
+<img height="32px" src="https://app.flowfuse.com/ff-logo--wordmark-caps--dark.png"></img>
+```
+
+You can replace the URL with your logo's URL or set it using the `msg.payload` as shown in examples given [documentation](https://dashboard.flowfuse.com/nodes/widgets/ui-template.html#page-name-app-bar-title).
+
+![Left Side Area](./images/logo-added-dashboard.png){data-zoomable}
+
+#### Hiding the Page Name in the Header
```

Comment:
I would put this section _before_ the examples so that, when users follow the examples this has already been actioned. Otherwise their output won't look the same as your screenshots above.